### DISCUSSION TITLE
文章發布流程

### DISCUSSION THREAD

**Discussion by yurenju** [lun-dao/LunDAO#35] [General]
想要來討論一下文章發表流程，我想先提出一個草案，大家再看看這樣適不適合，有沒有要刪減或新增的步驟。我的想法是剛開始的文章發布流程應該要簡單一些，隨著我們這個平台稍微跑了一段時間之後再來思考哪些增加。

1. 文章提案（選用）：如果想知道特定文章適不適合發表，直接在 Discord 或是 Github Discussion 詢問
2. 文章提交：採用 Pull Request 的方式提交文章，除了文章本體以外，PR 中應該要說明目標讀者，讓文字編輯可以把自己帶入目標讀者來協助編輯文章
3. 文章編輯：另外一個（或多個，自由參與）社群成員擔任文章編輯的角色，把自己當作是作者所設定的目標讀者，閱讀文章看有哪些需要修改的地方，與作者討論
4. 文章發表：合併之後自動發表
5. 獎勵：固定一段時間後比如說一個月或一季，看看用什麼適合的方式來讓大家討論哪篇文章對社群影響力最大。目前的想法是會有一個固定的文章獎勵跟加成的獎勵，比如說 100 LUN (固定) + 0-100 LUN (浮動)

想問問看大家覺得這個流程如何？有沒有什麼完全不同的流程或是其他想法？另外如果有些朋友有熟悉平方投票法也想了解一下這樣的情況是不是適合用平方投票法的地方。

**Comment by quesalltime** [comment]
目前對於文章的分級，已經有些討論，詳細可參考[#23](https://github.com/lun-dao/LunDAO/discussions/23)
有了guideline指標作為文章的分級參考之後，
下一步要考慮的應該就是由誰來審這件事。

「由誰審」這件事，可能有幾種做法

第一種：
由專案的初始參與人或是社群整體成員，選出大家認為qualified的審核委員，
再由審核委員對文章進行分級的認定與最終決斷。
至於審核委員的選出及任期方式，可再另外設計。

第二種：
由社群整體成員對文章進行分級的認定與最終決斷。

前者的好處是，文章分級的決定過程會相對有效率，
後者的好處是，讓社群整體成員對於LunDAO專案的核心目標，有更直接的參與過程，
兩者各有優劣。

**Reply by yurenju** [reply]
（我也把我寫的貼過來） https://github.com/lun-dao/LunDAO/discussions/23#discussioncomment-1921877

還有另外一個要考量的是「作者的人緣」，但這問題我也不知道是大是小，比如說有一個作者他人緣非常好，他的朋友如果持有代幣就都幫他投了，這樣也是讓整個社群投的問題之一。

**Comment by gasolin** [comment]
審核我覺得有意願的都可以審, 不用特別指定 (有的是審內容, 有的人在意 copy writing, 當然作者或評論者也可以 at 人來幫忙看)
覺得 ok 就在 github 上 review approve, 代表這篇我看過了覺得可以 merge
merge 前至少要有1~2個 approve,
最後當然是有 github merge 權限的人裁量什麼時候 merge

衡量 寫作/review 貢獻的部分 統一在 coordinape  https://github.com/lun-dao/LunDAO/discussions/26


若文章發布流程有共識了 也要把怎麼操作寫進編輯流程中

**Reply by yurenju** [reply]
我也覺得文章發布的討論得差不多了，我先來開個 issue，先把流程寫成一個 markdown，然後應該是 Docusaurus 的其中一頁。 #41

**Comment by johnson86tw** [comment]
> 5. 獎勵：固定一段時間後比如說一個月或一季，看看用什麼適合的方式來讓大家討論哪篇文章對社群影響力最大。目前的想法是會有一個固定的文章獎勵跟加成的獎勵，比如說 100 LUN (固定) + 0-100 LUN (浮動)

> 想問問看大家覺得這個流程如何？有沒有什麼完全不同的流程或是其他想法？另外如果有些朋友有熟悉平方投票法也想了解一下這樣的情況是不是適合用平方投票法的地方。

針對第 5 點獎勵的部分，提供一個引入平方投票法的方式：採用 [clr.fund](https://github.com/clrfund/monorepo) 來對文章進行捐款（投票）和獎勵分配。

clr.fund 可以像 gitcoin grants 一樣對專案進行「平方募資」，這裡的專案就改成作者的「文章」，然後 token 全部使用 LUN，運作方式大概會是這樣：
- 部署 clr.fund 合約
- LunDAO 送一筆 matching funds 到合約裡，例如 100 LUN
- 每季或每月舉辦一次募資活動
- 活動期間，社群成員使用 LUN 對「文章」進行捐款（投票）
- 活動結束後，文章作者可以獲得成員直接捐贈的 LUN，同時依照平方募資法的公式，從 matching pool 分配到另一筆 LUN 獎勵

這個做法會費一番功夫，但是它的確可以做到說，由社群成員（使用社群貨幣 LUN）投票決定哪個文章可以分配到最多獎勵。

**Reply by yurenju** [reply]
@chnejohnson 那他比起一般的每個人分配固定點數投票有什麼好處呢？

**Reply by gasolin** [reply]
@chnejohnson 獎勵方式感覺可以另外開一篇討論，讓這個討論串可以專注在文章發佈這塊 :)

**Reply by johnson86tw** [reply]
> @chnejohnson 那他比起一般的每個人分配固定點數投票有什麼好處呢？

如果是像總統杯黑客松那樣每人固定發 99 點，單純平方投票的話也是可以選出社群影響力最大的幾篇文章，然後給予 LUN 獎勵，這樣的話就是 DAO 依照投票表決結果，依照票選排名可以獲得不同等級的獎勵，這部分要再訂定獎勵辦法。

平方募資的話就不用訂定獎勵辦法，DAO 只需要思考要出多少資源到 matching pool，剩下就交給社群投票的結果，靠機制自然決定文章會分配到多少獎勵。

**Reply by yurenju** [reply]
@gasolin 說得對，再開另外一篇額外討論好了， @gasolin 幫忙起個頭好嗎？

**Reply by gasolin** [reply]
#40
